riscos_toolbox/gadgets/scrolllist.py

```diff
@@ -45,7 +45,7 @@ def delete_items(self, start, end):
 
     def get_selected(self, offset=-1):
         return swi.swi('Toolbox_ObjectMiscOp', 'IiIii;i',
-                       0, self.window.id, 16416, Scrollist.GetSelected, self.id, offset)
+                       0, self.window.id, 16416, ScrollList.GetSelected, self.id, offset)
```

Comment:
There's a duplicate argument in here, the old literal 16416 should have been removed when the ScrollList.GetSelected constant was added.